src/lib_sylt.rs

```diff
@@ -53,4 +53,37 @@ sylt_macro::extern_function!(
     },
 );
 
+sylt_macro::extern_function!(
+    "sylt::lib_sylt"
+    sin
+    [Value::Float(t)] -> Type::Float => {
+        Ok(Value::Float(t.sin()))
+    },
+);
+
+sylt_macro::extern_function!(
+    "sylt::lib_sylt"
+    cos
+    [Value::Float(t)] -> Type::Float => {
+        Ok(Value::Float(t.cos()))
+    },
+);
+
+
+#[sylt_macro::sylt_link(inf, "sylt::lib_sylt")]
+pub fn inf(values: &[Value], _typecheck: bool) -> Result<Value, RuntimeError> {
+    match values {
+        [x] => {
+            let t: Type = Type::from(&*x);
+            let x = x.clone();
+            Ok(Value::Iter(t, Rc::new(RefCell::new(move || Some(x.clone())))))
+        }
+        values => Err(RuntimeError::ExternTypeMismatch(
+            "push".to_string(),
+            values.iter().map(Type::from).collect(),
+        )),
+    }
+}
```

Comment:
From the name I assume this is supposed to be an infinite iterator, but when I test `example_game.sy` it runs for about one second before terminating.

Reply:
I've been made aware of the `break`. Disregard.

Reply:
I've been made aware of the `break` in the .sy. Disregard.